### ISSUE TITLE
[your-email] does not have storage.objects.list access to the Google Cloud Storage bucket

### ISSUE THREAD

**Issue by zhangshuoneu** [LightwheelAI/street-gaussians-ns#23]
Hi, Thanks for sharing the code! When I want to download the  waymo perception_v1.4.0 from you shared google buckets, I get error on download data. 
<img width="416" alt="gcs-bucket" src="https://github.com/LightwheelAI/street-gaussians-ns/assets/64781618/4ebc72a1-b4bf-4345-b4d2-bc42643f9599">
I am not very familiar with Google's buckets. Have you ever encountered similar problems or know how to solve them? Thank you very much

**Comment by HengDora** [comment]
Sign up [here](https://waymo.com/open/download/) might solve this problem.

**Comment by lolidrk** [comment]
> Sign up [here](https://waymo.com/open/download/) might solve this problem.

Yes I was facing the same problem, and you need to have an account on waymo in order to access the dataset. Once you're signed up, you can freely access it.
